### PR TITLE
Updates to TripleLift adapter

### PR DESCRIPTION
**Pull request by tiffwu**
Add support for dynamic floors and fix formatting for size parameter.

### CODE REVIEW

**Comment by mkendall07** [comment]
@prebid/triplelift please review.

@tiffwu thanks for the PR. Are you affiliated with TripleLift?

**Comment by tiffwu** [comment]
@mkendall07 Yep, I work with @mattjone on the Publisher Solutions Engineering team at TripleLift.

File: src/adapters/triplelift.js [hunk 1]
```diff
@@ -31,21 +31,24 @@ var TripleLiftAdapter = function TripleLiftAdapter() {
...
+
 
     //build our base tag, based on if we are http or https
     var tlURI = '//tlx.3lift.com/header/auction?';
     var tlCall = document.location.protocol + tlURI;
 
     tlCall = utils.tryAppendQueryString(tlCall, 'callback', 'pbjs.TLCB');
     tlCall = utils.tryAppendQueryString(tlCall, 'lib', 'prebid');
-    tlCall = utils.tryAppendQueryString(tlCall, 'lib', '0.5.0');
+    tlCall = utils.tryAppendQueryString(tlCall, 'v', '0.5.0');
```
Comment (mkendall07) [inline src/adapters/triplelift.js]: Is the version here for TripleLift or prebid.js? If prebid, then you can use the `$prebid.version$` macro to populate from the build process.

**Comment by mkendall07** [comment]
@tiffwu 
Looks good. Would you please add a unit tests coverage for these changes? Thanks

**Comment by tiffwu** [comment]
@mkendall07 Thanks for the review -- will add tests.

**Comment by tiffwu** [comment]
@mkendall07 We have a publisher client who would like these updates ASAP...is there any way we could merge this in for now while we work on the tests in parallel?

File: src/adapters/triplelift.js (hunk 1 above)
Comment (mkendall07) [inline src/adapters/triplelift.js]: @tiffwu didn't see a response here? Is this intended? Thanks

File: src/adapters/triplelift.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ var TripleLiftAdapter = function TripleLiftAdapter() {
 
     tlCall = utils.tryAppendQueryString(tlCall, 'callback', 'pbjs.TLCB');
     tlCall = utils.tryAppendQueryString(tlCall, 'lib', 'prebid');
-    tlCall = utils.tryAppendQueryString(tlCall, 'v', '0.5.0');
+    tlCall = utils.tryAppendQueryString(tlCall, 'v', $prebid.version$);
```
Comment (mkendall07) [inline src/adapters/triplelift.js]: Thanks. Should be a string. run `gulp serve` to see the build errors.

**Comment by mkendall07** [comment]
LGTM. Just add quotes around `$prebid.version$` and we can merge.  Please open a new PR with unit tests. Thanks

**Comment by mkendall07** [comment]
@tiffwu ^^

**Comment by tiffwu** [comment]
Much appreciated, @mkendall07!

**Pull request merged**